deployments/vanilla/terraform/Makefile

```diff
@@ -0,0 +1,42 @@
+SHELL := /bin/bash # Use bash syntax
+
+deploy: init deploy_vpc deploy_eks_blueprints deploy_eks_blueprints_k8s_addons deploy_kubeflow_components
+
+destroy: deploy_kubeflow_components destroy_eks_blueprints_k8s_addons destroy_eks_blueprints destroy_vpc
+	terraform destroy -auto-approve
+
+init:
+	terraform init
+
+
+deploy_vpc:
+	terraform apply -target="module.vpc" -auto-approve
+
+deploy_eks_blueprints:
```

Comment:
for vpc and eks is it better to call create and delete

how about just create-eks-cluster

Reply:
done